### PR TITLE
Template for Case Studies

### LINKED ISSUE #1473: Create a series of template to facilitate contributions
### Summary

* As the book has evolved, it's content has also started to develop in modular size that makes the template outdated
* We need to create new templates to assist our contributors create their contents

### What needs to be done?

These templates should be crowdsourced with the help of @paulowoicho:

- [x] Template for a landing page of the chapter
- [x] Template for a small subchapter 
- [x] Case studies for reproducibility
- [x] Case studies for communication
- [x] Case studies for collaboration
- [x] Case studies for ethical research
- [x] Case studies for project design
- [ ] Personal interview or a personal story (generally can be added as a last subchapter of a chapter)

A few more things to think about:

- [x] When creating sections in a template, add prompts to help authors brainstorm what they should think about their content
- [x] How they can cross link subchapters if any
- [x] Maybe add the short consistency checklist as comment at the end of these templates to ensure that they can review their contribution before sending a PR for review
- [x] Take time to consider if specific guide will require some specific prompts or one template will fit all guides

### Who can help?

* Anyone who is interested in helping develop this

---

### PR DESCRIPTION
**Pull request by paulowoicho**
### Summary

Fixes #1473

In this PR, I create a template for writing case studies in _The Turing Way_

### List of changes proposed in this PR (pull-request)

* New template for writing Turing Way Case Studies

### What should a reviewer concentrate their feedback on?

- [ ] Are the prompts useful and adequate enough to help authors write their content?
- [ ] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: book/templates/case-study-template/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Welcome
+
+Thank you for deciding to write a case study for _The Turing Way_ book!
```
Comment (malvikasharan) [inline book/templates/case-study-template/README.md:3]:
```suggestion
Thank you for contributing to _The Turing Way_ book by writing a case study!

_The Turing Way_ chapters are co-authored by people from different research backgrounds and domain expertise.
 We do not aim to create domain-specific resources but provide a central collection of best practices, tools and recommendations in data science that are transferable across different disciplines.
 Nevertheless, we recognise that to allow readers to gain an in-depth understanding of new topics we need to provide some examples and case studies that they can identify with.
Therefore, community-contributed case studies are highly beneficial for our community members who would like to see some relatable examples where these skills are applied.
```

File: book/templates/case-study-template/README.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+│   │
+│   └───case-studies
+│       │   statistical-methods-manuscript.md <---- (Statistical Manuscripts Case Study)
+|       |   Case Study 2.md
+|       |   ...
+│   
+└───project-design <---- (Folder for the Guide for Project Design)
+    │   project-design.md
+    │   ...
+```
```
Comment (malvikasharan) [inline book/templates/case-study-template/README.md:33]:
```suggestion

```
book/website
│
└───reproducible-research <---- (Folder for the Guide to Reproducible Research)
│   └───binderhub
|   └─── case-studies  <---- (folder containing subchapter for case studies the Guide to Reproducible Research)
│       │   statistical-methods-manuscript.md <---- (Statistical Manuscripts Case Study)
|       |   example-case-study-file.md
|   |   ...
│   │   case-studies.md <---- (Landing page for Case Studies chapter)
|   |   ...
│   
└───project-design <---- (Folder for the Guide for Project Design)
    │   project-design.md
    │   ...
```
```

**Comment by malvikasharan** [comment]
This looks fantastic @paulowoicho. I added some minor updates.

**Comment by paulowoicho** [comment]
Thanks for the review! I have revised the PR with the new suggestions

**Review by malvikasharan — APPROVED** [review]
Looks good to me. Please merge it when you are ready.

**Pull request merged**